gui/widgets/qopenglmousetracker.cpp

```diff
@@ -14,19 +14,24 @@ QOpenGLMouseTracker::QOpenGLMouseTracker(QWidget *parent, float lineWidth,
       m_lineSelectPrecision(lineSelectPrecision),
       m_lineWidthSelectedSubmesh(lineWidthSelectedSubmesh),
       m_selectedObjectColor(selectedObjectColor),
-      m_camera(cameraFOV, static_cast<float>(size().width()),
-               static_cast<float>(size().height()), cameraNearZ, cameraFarZ),
+      m_camera(std::make_shared<rendering::Camera>(
+          cameraFOV, static_cast<float>(size().width()),
+          static_cast<float>(size().height()), cameraNearZ, cameraFarZ)),
       m_frameRate(frameRate),
       m_backgroundColor(QWidget::palette().color(QWidget::backgroundRole())),
-      m_lastColour(QWidget::palette().color(QWidget::backgroundRole()).rgb()) {}
+      m_lastColour(QWidget::palette().color(QWidget::backgroundRole()).rgb()) {
+
+  // A default camera is added.
+  m_sceneGraph->add(m_camera);
+}
 
 std::shared_ptr<rendering::ClippingPlane>
 QOpenGLMouseTracker::BuildClippingPlane(
     GLfloat a, GLfloat b, GLfloat c, GLfloat d, bool active,
-    std::shared_ptr<rendering::Node> parent) {
+    bool localFrameCoord, std::shared_ptr<rendering::Node> parent) {
```

Comment:
`std::shared_ptr<rendering::Node>&`: we're not taking ownership of parent, just modifying it